cmd/mirror-main.go

```diff
@@ -608,7 +620,7 @@ func runMirror(srcURL, dstURL string, ctx *cli.Context) *probe.Error {
 				}
 				// Copy policy rules from source to dest if flag is activated
 				if ctx.Bool("a") {
-					err := copyBucketPolicies(srcClt, dstClt, ctx.Bool("force"))
+					err := copyBucketPolicies(srcClt, dstClt, ctx.Bool("overwrite"))
```

Comment:
We should use `isOverwrite` here, not `ctx.Bool("overwrite")`

Reply:
why would we do that? the code is kept similar to before wherever `force` was used now uses `overwrite`

Reply:
i see now.

Reply:
> why would we do that? the code is kept similar to before wherever force was used now uses overwrite

Just to have full backward compatibility, isOverwrite is just calculated above.